sdk/confidentialledger/confidential-ledger-rest/review/confidential-ledger.api.md

```diff
@@ -7,195 +7,208 @@
 import { Client } from '@azure-rest/core-client';
 import { ClientOptions } from '@azure-rest/core-client';
 import { HttpResponse } from '@azure-rest/core-client';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PathUncheckedResponse } from '@azure-rest/core-client';
 import { RawHttpHeaders } from '@azure/core-rest-pipeline';
 import { RequestParameters } from '@azure-rest/core-client';
+import { StreamableMethod } from '@azure-rest/core-client';
 import { TokenCredential } from '@azure/core-auth';
 
 // @public (undocumented)
-function ConfidentialLedger(ledgerBaseUrl: string, ledgerTlsCertificate: string, options?: ClientOptions): ConfidentialLedgerRestClient;
-
-// @public (undocumented)
-function ConfidentialLedger(ledgerBaseUrl: string, ledgerTlsCertificate: string, credentials: TokenCredential, options?: ClientOptions): ConfidentialLedgerRestClient;
-export default ConfidentialLedger;
-
-// @public (undocumented)
-export interface ConfidentialLedgerEnclaves {
-    currentNodeId: string;
-    enclaveQuotes: EnclaveQuotesDictionary;
+export interface CollectionOutput {
+    // (undocumented)
+    collectionId: string;
 }
 
 // @public (undocumented)
-export interface ConfidentialLedgerError {
-    error?: ConfidentialLedgerErrorBody;
+export interface CollectionsOutput {
+    collections: Array<CollectionOutput>;
 }
 
 // @public (undocumented)
-export interface ConfidentialLedgerErrorBody {
-    code?: string;
-    innerError?: ConfidentialLedgerErrorBody;
-    message?: string;
-}
+function ConfidentialLedger(ledgerBaseUrl: string, ledgerTlsCertificate: string, options?: ClientOptions): ConfidentialLedgerClient;
 
 // @public (undocumented)
-export type ConfidentialLedgerQueryState = "Loading" | "Ready";
+function ConfidentialLedger(ledgerBaseUrl: string, ledgerTlsCertificate: string, credentials: TokenCredential, options?: ClientOptions): ConfidentialLedgerClient;
```

Comment:
would it make sense to call the second parameter `ledgerIdentityCertificate` do you think that makes sense in the context of Confidential Ledger?

Since the client can also authenticate with TLS certificates that are different to this cert that is passed to the factory function I'd like to avoid customers confusing `options.tlsSettings` with this certificate